README.md

```diff
@@ -2,66 +2,61 @@
 
 ![Build Status](https://github.com/spice-labs-inc/goatrodeo/actions/workflows/scala.yml/badge.svg)
 
-The Software Supply Chain is a big gnarly mess of unknowns... and it's
-baked right into all the systems you and your company build an run. 😱
```

Comment:
I like having this sentence at the top to set up the "why".  Could we keep it, or something like it?